addons/analytic/views/analytic_line_views.xml

```diff
@@ -8,9 +8,9 @@
                 <field name="company_id" invisible="1"/>
                 <field name="product_uom_category_id" invisible="1"/>
                 <field name="date" optional="show"/>
-                <field name="name"/>
-                <field name="account_id"/>
-                <field name="plan_id"/>
+                <field name="name" optional="show"/>
+                <field name="account_id" optional="show"/>
+                <field name="plan_id" optional="show"/>
```

Comment:
OK for me if @tsb-odoo or @gawa-odoo is ok.
Seems weird to do this on the default view, should we not have a separate view for MRP analytic margin report?
Many users might activate these if they use analytic accounts for anything else.

Reply:
Indeed, it seems weird to do so for all analytic lines. 
If someone hides it for the view coming from 'Gross margin', it does not mean he wants to hide for the general view of analytic lines. (And I find especially weird if you hide the analytic account for the general view where we group by analytic account by default)